### ISSUE TITLE
Spatialite: allow editing of geometery of features

### ISSUE THREAD

**Issue by brentfraser** [geopaparazzi/geopaparazzi#624]
Currently available:
- create new feature 
- delete feature

But editing the geometry is not available:
- move existing points
- move existing points on existing lines and polygon outlines

**Comment by moovida** [comment]
Nope, that has been a design choice. It would have been too difficult to implement with the framework at hand. There is an undo and for polygons there is a cut-extend tool. 

I am not sure what we should do with issues like these. They will likely not be implemented. What do you think?

**Comment by brentfraser** [comment]
I expected this.  In case users have expectation about functionality (especially if it was in a prior version) these issues should be filed and closed so if they search the issues they will find them.  And I'm struggling to decide what to say in the manual about functions that were available, but are not longer.

**Comment by moovida** [comment]
Ok, I understand.
But regarding editing only one functionality is missing, and I highly doubt anyone was using it.
Or do you have doubts?

**Comment by brentfraser** [comment]
I agree, but then we only recently documented it.  Editing geometry in the field, especially on a phone, is hard (and likely no one was doing it).
